<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbPopuniIzvore">
       <property name="text">
        <string>Popuni izvore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Izvori:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lwIzvori"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ukupna zapremina izvora:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leUkZapreminaIzvora">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pbZapocniTransfer">
       <property name="text">
        <string>Zapocni transfer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ukupna zapremina rezervoara:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leUkZapreminaRezervoara"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tekuca zapremina rezervoara:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTekZapRezervoara">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Gubitak:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leGubitak">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>450</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
